cmd/podman/common/completion.go

```diff
@@ -1471,8 +1471,14 @@ func AutocompleteClone(cmd *cobra.Command, args []string, toComplete string) ([]
 	}
 	switch len(args) {
 	case 0:
+		if strings.Contains(cmd.Example, " pod ") { // needs to be " pod " to exclude 'podman'
```

Comment:
This is a very awkward and can break at any time. Please use cmd.Parent().Name() == "pod"

Reply:
ah yes, thanks I was looking for something more concrete here.